src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -268,6 +320,56 @@ private VariantContext calculateGenotypes(VariantContext vc){
         return genotypingEngine.calculateGenotypes(vc, model, null);
     }
 
+    /**
+     * Remove INFO field annotations if no alternate alleles
+     *
+     * @param vc    the variant context
+     * @return variant context with the INFO field annotations removed if no alternate alleles
+     */
+    private VariantContext removeInfoAnnotationsIfNoAltAllele(final VariantContext vc)  {
```

Comment:
this is unused

Reply:
hm. @ldgauthier I think it was a mistake on my part that I included this method in this PR when I ported these changes from GATK3, since this code doesn't appear to be explicitly dependent on `includeNonVariantSites`. However, is it correct that GATK doesn't calls this ?

Reply:
I would rather call that in GATK4 as well.  If it's not variant then we don't need annotations for filtering downstream.